core/matching/orderbook_test.go

```diff
@@ -3326,6 +3326,10 @@ func TestOrderBook_AuctionUncrossWashTrades(t *testing.T) {
 
 	// Get indicative trades
 	trades, err := book.ob.GetIndicativeTrades()
+	// because the trades are generated when calling LeaveAuction, the aggressor will be unspecified.
```

Comment:
Why are we doing this and not just changing the assert to be:
```
assert.Equal(t, uncrossedOrders[0].Trades[0].Aggressor, vega.Side_SIDE_UNSPECIFIED)
```

Reply:
Because copying the loop from the `orderbook.go` file was quicker :smile: